src/ngraph/frontend/onnx_import/op/eye_like.cpp

```diff
@@ -0,0 +1,87 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <onnx-ml.pb.h>
```

Comment:
We should not hardcode `-ml` here if possible.

Reply:
The whole `onnx_importer` have `-ml` hardcoded I think this is a topic for another story to change it.